monkestation/code/modules/power/rbmk/rbmk_meltdown_fallout.dm

```diff
@@ -0,0 +1,117 @@
+#define SLUDGE_SHORT_RANGE 5
+#define SLUDGE_LONG_RANGE 60
+
+/obj/effect/decal/nuclear_waste
+	name = "Plutonium sludge"
+	desc = "A writhing pool of heavily irradiated, spent reactor fuel. A shovel should clear it up, though you probably shouldn't step through this..."
+	icon = 'monkestation/icons/obj/machinery/reactor_parts.dmi'
+	icon_state = "nuclearwaste"
+	alpha = 150
+	light_color = LIGHT_COLOR_CYAN
+	color = "#ff9eff"
+
+/obj/effect/decal/nuclear_waste/Initialize()
+	. = ..()
+	set_light(3)
+
+/// The one that actually does the irradiating. This is to avoid every bit of sludge PROCESSING
+/obj/effect/decal/nuclear_waste/epicenter
+	name = "Dense nuclear sludge"
+
+/// Clean way of spawning nuclear gunk after a reactor core meltdown.
+/obj/effect/landmark/nuclear_waste_spawner
+	name = "Nuclear waste spawner"
+
+/obj/effect/landmark/nuclear_waste_spawner/proc/fire()
```

Comment:
So you're going to scream at me.
But I think moving it back to a variable and basing the range off the total 'power' in the reactor at the time of detonation would be good. Add a little random element to it too.

Currently it ALWAYS reaches the bar and kitchen, never further away and never a shorter range.
Have a really nasty blast blow through the station and leave residue everywhere, while a new engineer that just fucks up a bit get a localized dusting.